### ISSUE TITLE
Update links at Readme for codesandbox demos

### ISSUE THREAD

**Issue by twbagustin** [darrenjennings/vue-autosuggest#147]
- `vue-autosuggest` version: 2.0.4
- `vue` version: 2.6.9

Relevant code or config
```javascript
  "dependencies": {
    "axios": "0.17.1",
    "vue": "2.6.9",
    "vue-autosuggest": "^2.0.4"
  }
```

What you did:
Used vue 2.6.9, and vue-autosuggest 2.0.4 at https://codesandbox.io/s/mjqrk7v2rx

What happened:
Apparently `this.fetchResults` is not being called

Reproduction repository:
Just head over to your "Api Fetching suggestions with Multiple sections" Codesandbox demo and you will see the issue: https://codesandbox.io/s/mjqrk7v2rx, if it helps I'm on google chrome Version 77

Problem description:
With v.1.8.3 works! Apparently issue might have been introduced at v2.0?
```javascript
  "dependencies": {
    "axios": "0.17.1",
    "vue": "2.6.9",
    "vue-autosuggest": "1.8.3"
  }
```
Suggested solution:
-

**Comment by twbagustin** [comment]
Update, problem is the readme wasn't updated after v2.0 so used a "deprecated" codesandbox example in v2.0.0 you mention a fix for #95 there you posted an updated snippet https://codesandbox.io/s/x9z1lo20mo

**Comment by darrenjennings** [comment]
Thanks for the issue! These definitely needed some love.

Check out the latest links and let me know if you run into any issues:
https://github.com/darrenjennings/vue-autosuggest/commit/c2f7e88d6cdc6ce476babfb4b8a76e0b96b1a2ef
